frontend/src/admin/historical_data_entry/components/HistoricalDataTable.js

```diff
@@ -76,7 +76,7 @@ const HistoricalDataTable = (props) => {
         return -1; // this is to fix sorting (value can't be negative)
       }
 
-      return parseFloat(item.totalValue);
+      return parseFloat(item.totalValue).toFixed(2);
```

Comment:
There aren't any business rules/legislation on how we round is there?

Reply:
what do you mean? I used 2 decimal places for the reason that that's what's closes to what the user will see in the table.

Reply:
Financial systems often have rules around rounding up and rounding down - I'm guessing it's not applicable but given that there's a financial impact with any numbers we're dealing with it may be applicable.

Reply:
I'll have to confirm this with Justin